### ISSUE TITLE
Infinity loop

### ISSUE THREAD

**Issue by Falci** [buglss/node-red-clone-inject-with-cronti#18]
Not sure what's going on, but I tried a "weekdays" cron and it fires in a infinity loop.

If I select "Saturday", everything works as expected. Today is Saturday, not sure if it is relevant.

![image](https://github.com/buglss/node-red-crontinject/assets/1330573/3b46d451-f163-4ddf-b375-5c052fbaf532)

<details>
<summary>Code</summary>

```json
[
    {
        "id": "04313c2024c2b622",
        "type": "crontinject",
        "z": "bbedaa23cd3b34e8",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "00 00 * * 1,2,3,4,5",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "",
        "payloadType": "date",
        "crontiMethod": "",
        "crontiArgs": "[]",
        "inputs": 0,
        "hasButton": true,
        "x": 1910,
        "y": 320,
        "wires": [
            [
                "2cc6745b9ab03abe"
            ]
        ]
    },
    {
        "id": "2cc6745b9ab03abe",
        "type": "debug",
        "z": "bbedaa23cd3b34e8",
        "name": "debug cron",
        "active": true,
        "tosidebar": true,
        "console": false,
        "tostatus": false,
        "complete": "true",
        "targetType": "full",
        "statusVal": "",
        "statusType": "auto",
        "x": 2330,
        "y": 320,
        "wires": []
    }
]
```
</details>

**Comment by buglss** [comment]
@Falci I couldn't observe any problems when I tried it. Can you write down the edits you made regarding this node, one by one? A setting you previously made may not have been cleared completely.

**Comment by buglss** [comment]
Changed has been stora method by repeaters. (https://github.com/buglss/node-red-crontinject/commit/904352e7a25923e701ef878fd0216109aa934b55)

**Comment by rogerzhang214** [comment]
Had the same issue even with the new 3.0.1 update. Steps to reproduce:
1. select "at a specific time"
2. choose a time, for instance I chose the next minute for testing purposes
3. for the day of week option I selected only thursday which is today
4. deploy
When the timer hits the next minute, the node executes infinite times non stop
FYI, I also tried selecting all the days of week and it works just fine.
